build/configs/rtl8721csm/security_hal_test_tz/defconfig

```diff
@@ -0,0 +1,1298 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# TinyAra Configuration
+#
+
+#
```

Comment:
How about merging this into security_hal_test which is already existed? I don't think we need to provide two defconfig for security hal test.

Reply:
Yes, we can. This security_hal_test_tz is the correct one. I will delete the previous security_hal_test.
Because eventually Trust Zone have to be enable.